packages/api-client/src/components/Server/ServerVariablesSelect.vue

```diff
@@ -0,0 +1,52 @@
+<script setup lang="ts">
+import {
+  ScalarButton,
+  ScalarIcon,
+  ScalarListbox,
+  type ScalarListboxOption,
+} from '@scalar/components'
+import { computed } from 'vue'
+
+const props = defineProps<{
+  enum: string[]
+  value: string
+  controls?: string
+}>()
+
+const emit = defineEmits<{
+  (e: 'change', v: string): void
+}>()
+
+const options = computed<ScalarListboxOption[]>(() =>
+  props.enum.map((s) => ({ id: s, label: s })),
+)
+
+const selected = computed<ScalarListboxOption | undefined>({
+  get: () => options.value.find((opt) => opt.id === props.value),
+  set: (opt?: ScalarListboxOption) => emit('change', opt?.id ?? ''),
+})
+</script>
+<template>
+  <ScalarListbox
+    v-model="selected"
+    :options="options">
+    <ScalarButton
+      :aria-controls="controls"
+      class="h-8 p-0 py-1.5"
+      fullWidth
```

Comment:
You can just put it in a flexbox to achieve the same thing (or add `w-full` to the classes)